### PR TITLE
fix building WFE Handlebars

### PR DESCRIPTION
**Pull request by blizzz**
I missed that in #15488 (probably due to browser caching :-/ )

Currently, loading a workflow app's settings would log the error `Missing helper: "selectItem" and thus could not function (is not rendered properly) and building would fail with ` Error: You specified knownHelpersOnly, but used the unknown helper selectItem - 16:32`. That's all addressed now.

### CODE REVIEW

File: apps/workflowengine/src/admin.js [hunk 1]
```diff
@@ -310,6 +294,7 @@ import OperationsTemplate from './templates/operations.handlebars';
 				this.model.set(key, value);
 			},
 			render: function() {
+				console.warn('WFE render');
```
Comment (blizzz) [inline apps/workflowengine/src/admin.js]: oh, let me remove that

File: apps/workflowengine/webpack.js [hunk 2]
```diff
@@ -31,11 +31,17 @@ module.exports = {
 			},
 			{
 				test: /\.handlebars$/,
-				loader: "handlebars-loader"
-			}
+				loader: "handlebars-loader",
+				options: {
+					helperDirs: path.join(__dirname, 'src/hbs_helpers'),
+					precompileOptions: {
+						knownHelpersOnly: false,
```
Comment (ChristophWurst) [inline apps/workflowengine/webpack.js]: this seems not be necessary with the `helperDirs` options. I removed it locally and the build and WFE did not show any errors

**Review by ChristophWurst — COMMENTED** [review]
Otherwise this looks great :+1:

File: apps/workflowengine/webpack.js (hunk 2 above)
Comment (blizzz) [reply apps/workflowengine/webpack.js]: indeed

**Review by rullzer — APPROVED** [review]
Makes sense

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**